presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java

```diff
@@ -70,7 +72,7 @@ public void testAcos()
     {
         for (double doubleValue : DOUBLE_VALUES) {
             assertFunction("acos(" + doubleValue + ")", DOUBLE, Math.acos(doubleValue));
-            assertFunction("acos(FLOAT '" + (float) doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
+            assertFunction("acos(FLOAT '" + doubleValue + "')", DOUBLE, Math.acos((float) doubleValue));
```

Comment:
Are you going to change these? I'd imagine that trig functions would be very susceptible to double/float changes